module/account/src/Domain/Event/Role/RolePrivilegesChangedEvent.php

```diff
@@ -33,18 +42,28 @@ class RolePrivilegesChangedEvent implements DomainEventInterface
     private $to;
 
     /**
+     * @param RoleId      $id
      * @param Privilege[] $from
      * @param Privilege[] $to
      */
-    public function __construct(array $from, array $to)
+    public function __construct(RoleId $id, array $from, array $to)
```

Comment:
On construct you require `RoleId` and in `getAggregateId` u allow to return `AbstractId`. Why u don't set `AbstractId` in constructor?
PHP 7.4 allow to set in interface AbstractId as return type and in base class u can set RoleId as return type. PHP will compile it without error :)
When u setup constructor as `__construct(AbstractId $id, array $from, array $to)` u don't need to setup constructor that many times :)